admin/faq.php

```diff
@@ -29,11 +29,11 @@
 
      '<p><strong>' . __( 'How long should a backup take?', 'backupwordpress' ) . '</strong></p>' .
 
-     '<p>' . __( 'Unless your site is very large (many gigabytes) it should only take a few minutes to perform a back up, if your back up has been running for longer than an hour it\'s safe to assume that something has gone wrong, try de-activating and re-activating the plugin, if it keeps happening, contact support.', 'backupwordpress' ) . '</p>' .
+     '<p>' . __( 'Unless your site is very large (many gigabytes) it should only take a few minutes to perform a back up. If your back up has been running for longer than an hour, it\'s safe to assume that something has gone wrong. Try de-activating and re-activating the plugin. If it keeps happening, contact support.', 'backupwordpress' ) . '</p>' .
```

Comment:
should actually be `backup` in one word, if you'd like to fix that in the same PR